x/bank/wire.go

```diff
@@ -9,5 +10,8 @@ func RegisterWire(cdc *wire.Codec) {
 	// TODO include option to always include prefix bytes.
 	cdc.RegisterConcrete(SendMsg{}, "cosmos-sdk/SendMsg", nil)
 	cdc.RegisterConcrete(IssueMsg{}, "cosmos-sdk/IssueMsg", nil)
+
+	cdc.RegisterInterface((*sdk.Msg)(nil), nil)
```

Comment:
do we really want this here ? presumably its ok to register the same interface multiple times ?

Reply:
......... it should not be..... looking now

Reply:
bucky you're looking at old code - there were a couple of iterations in the refactoring

Reply:
This is now only registered one time at the application level and then propagated to the all commands as needed

Reply:
yep sorry I was reviewing one commit at a time!